### ISSUE TITLE
Create a user overview tab on account page

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#13136]
### Description
Many platforms provide various user information on a single screen before moving to editing. We could have information about our activities within Mautic on this screen, as well as our account email, etc.

### Impact
By consolidating key user information and activity details onto a single screen, users would benefit from a streamlined and user-friendly interface, enabling them to quickly access pertinent data and make informed decisions. 

This enhancement would improve the platform's usability and accessibility, as screen readers could see the information instead of dealing with editing fields directly.

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by RCheesley** [comment]
Still relevant

**Comment by andersonjeccel** [comment]
We do not have current functionalities needing an overview tab. It could improve the user experience as seen in https://github.com/mautic/mautic/pull/13756 but now it has low priority.
